packages/schemas/alterations/next-1695198741-remove-m2m-app-admin-access-switch.ts

```diff
@@ -0,0 +1,259 @@
+import { generateStandardId } from '@logto/shared/universal';
+import { deduplicate } from '@silverhand/essentials';
+import { sql } from 'slonik';
+
+import type { AlterationScript } from '../lib/types/alteration.js';
+
+enum InternalRole {
+  Admin = '#internal:admin',
+}
+
+enum RoleType {
+  User = 'User',
+  MachineToMachine = 'MachineToMachine',
+}
+
+enum PredefinedScope {
+  All = 'all',
+}
+
+const managementApiResourceIndicatorPrefix = 'https://';
+const managementApiResourceIndicatorSuffix = '.logto.app/api';
+const getManagementApiResourceIndicator = (tenantId: string) =>
+  `${managementApiResourceIndicatorPrefix}${tenantId}${managementApiResourceIndicatorSuffix}`;
+
+const managementApiAccessRoleName = 'Management API Access';
+const managementApiAccessRoleDescription = 'Management API Access';
+
+const alteration: AlterationScript = {
+  up: async (pool) => {
+    /**
+     * Step 1
+     * Get all internal admin roles.
+     * Notice that in our case: each internal admin role has only one scope (`PredefinedScope.All`), and each tenant has only one internal admin role.
+     */
+    const { rows: userAssignedInternalManagementApiRoles } = await pool.query<{
+      roleId: string;
+      tenantId: string;
+      scopeId: string;
+    }>(sql`
+      select * from (select roles.id as "role_id", roles.tenant_id as "tenant_id", scopes.id as "scope_id", resources.id as "resource_id", resources.indicator as "indicator" from roles join roles_scopes on roles_scopes.role_id = roles.id and roles_scopes.tenant_id = roles.tenant_id join scopes on scopes.id = roles_scopes.scope_id and scopes.tenant_id = roles_scopes.tenant_id join resources on resources.id = scopes.resource_id and resources.tenant_id = scopes.tenant_id
```

Comment:
Could you format this sql, hard to read?

Reply:
Refactored, also added comments for better readablity.